superset/security.py

```diff
@@ -262,27 +262,17 @@ def accessible_by_user(self, database, datasource_names, schema=None):
             full_names = {d.full_name for d in user_datasources}
             return [d for d in datasource_names if d in full_names]
 
-    def merge_perm(self, permission_name, view_menu_name):
```

Comment:
For backwards compatibility, we should just call `self.add_permission_view_menu` here and maybe add a deprecation warning.